src/ngx_http_upstream_jdomain_module.c

```diff
@@ -379,6 +396,7 @@ ngx_http_upstream_jdomain_create_instance(ngx_conf_t *cf, ngx_http_upstream_jdom
 	instance->conf.interval = 1;
 	instance->conf.max_ips = 4;
 	instance->conf.port = NGX_JDOMAIN_DEFAULT_PORT;
+	instance->conf.ipver = NGX_JDOMAIN_DEFAULT_IPVER;
```

Comment:
Following suggestion for line 581, also since it's a slight change in semantic, we should change the assignment:

```suggestion
	instance->conf.addr_family = AF_UNSPEC;
```